app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -87,7 +88,7 @@ class BackgroundServices(
         emptyList()
     }
 
-    private val defaultDeviceName = Build.MANUFACTURER + " " + Build.MODEL
+    private val defaultDeviceName = context.getString(R.string.app_name) + " on " + Build.MANUFACTURER + " " + Build.MODEL
```

Comment:
Since we're introducing a string here ("on") we need to make sure that it's being localized properly.

It might make more sense to have a string resource that has placeholders for the template we want. [Here's an example](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/values/strings.xml#L613) of the a string resource that has placeholders; we could add three of them:
1. The application name.
2. The manufacturer.
3. The model number.

Could you also add a comment similar to the other strings resources? It's for our localizers to understand the context of where the string is being used so that it can be translated it correctly.